specs/management.yml

```diff
@@ -12226,6 +12243,11 @@ definitions:
     properties:
       appData:
         $ref: '#/definitions/tags'
+      cost:
```

Comment:
cost should be required in edge router update. Unlike in the fabric, fingerprint should not be updateable here, since there's a separate process for managing enrollments/fingerprints that we don't want to bypass.

Reply:
added